lib/stdlib/src/filename.erl

```diff
@@ -34,7 +34,8 @@
 -export([absname/1, absname/2, absname_join/2, 
 	 basename/1, basename/2, dirname/1,
 	 extension/1, join/1, join/2, pathtype/1,
-	 rootname/1, rootname/2, split/1, nativename/1]).
+	 rootname/1, rootname/2, split/1, nativename/1,
+         safe_relative_path/1]).
```

Comment:
It is is misaligned in a diff, but not in an editor looking at the source code.

We have changed the rules for indentation. The recommendation for new or changed code is to indent using only spaces. (We have no intention of converting all tabs to spaces, as that should create an enormous diff.)

I have pushed a minor correction, eliminating the tab in the line I edited.